### PR TITLE
Created WordCountContainer

### PR DESCRIPTION
**Pull request by Kyleswillard**
# Pull Request Template
## Description
Created, styled and rendered the WordCountContainer on the NewParentDashboard. 
Add a summary that address the following:

- motivation/purpose for the feature
Our Team was working on the Average Word Count Card, and as such it needed a container to be rendered in, so this design, and the code was put together to ensure that this had a home. 
- What were those changes
 Created new component, and styled it, rendered it on new parent dashboard
- What issues were fixed
New component, gave soon to be endpoint a graphical representation
- Provide the link to the user story it is referencing in Trello board or other supporting document
https://trello.com/c/0YfcYmim
https://www.loom.com/share/91f661a96fbf41f1a1393416e6a042fa

## Commits Checklist

- [X] commits have clear title names
- [X ] commit has descriptive message of what has changed
- [X ] more small commits than less big commits

## Code Checklist

- [X ] code is formatted appropriately
- [X ] code meets DRY standards
- [X ] no dead code (check for logs and print statements)
- [ X] no TODOS instead necessary comments
- [ X] code follows Story Squad standards for:
  - Language
  - Framework
  - Libraries

### PRs needs to be reviewed by at least 2 team members

- TPL is the 3rd reviewer
- Lots of comments and suggestions, please!

[Pull Request Rubric](https://www.notion.so/1fc04e4fedeb429ba873b7c68d281707?v=74054da7991341c0bf970f39410c43da)

### CODE REVIEW

**Review by casey-dietz — APPROVED** [review]
The container looks great! Code is clean and concise. The description you wrote was very helpful on what was done. The loom video did not have any audio

**Comment by Kyleswillard** [comment]
> The container looks great! Code is clean and concise. The description you wrote was very helpful on what was done. The loom video did not have any audio

Thank you for the heads up. Not quite sure why. I will check on that.

File: src/components/pages/WordCountContainer/WordCountContainer.js [hunk 1]
```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+import './WordCountContainer.styles.css';
+// Documentation: This component is to show the average endpoint for the specified child. This component is rendered & styled on the NewParentDashboard.
+
+const WordCountContainer = () => {
+  return (
+    <div>
+      <div className="word-count-container">
```
Comment (tonysorensen) [inline src/components/pages/WordCountContainer/WordCountContainer.js:8]: Nice job following the html class naming convention. It looks weird to me seeing hyphens but you follow the convention consistently and it makes it very clear what you are looking at. Semantic naming like this makes code much easier to read.

File: src/components/pages/WordCountContainer/WordCountContainer.styles.css [hunk 2]
```diff
@@ -0,0 +1,22 @@
+.word-count-container {
+  margin: 0 auto;
+  width: 400px;
```
Comment (tonysorensen) [inline src/components/pages/WordCountContainer/WordCountContainer.styles.css:3]: Try to use responsive units whenever and wherever possible. Percents can be tricky but scale nicely when the app is viewed on different devices. One method is to set a base font size of 62.5% for all text in the HTML, this allows for easy computation of ems as responsive unit for things like height and width. Look into it for future projects!

**Review by tonysorensen — APPROVED** [review]
Approved, nice small changes and very easy to follow what was done.

**Pull request merged**